addons/website/static/src/less/website.ui.less

```diff
@@ -4,14 +4,16 @@
 
 // LAYOUTING
 body.o_connected_user {
-    padding-top: @odoo-navbar-height!important;
+    padding-top: @odoo-navbar-height - 2!important;
```

Comment:
@qsm-odoo , unfortunatly in [navbar.less](https://github.com/odoo/odoo/blob/10.0/addons/web/static/src/less/navbar.less#L12) we compensate the navbar border to set the height of tons of elements ( > ul > li > a, o_menu_systray....)

I dont like the arbitrary `-2` that is  in my code, but I don't like the idea  to overwrite all those rules nether :/

What you suggest?